source/src/notebook/healthcare-and-life-sciences/molecular-unfolding/molecular_unfolding.ipynb

```diff
@@ -0,0 +1,891 @@
+{
```

Comment:
<p>Please double check if the user can see all pictures in Notebook. The picture can not be shown in ReviewNB.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/awslabs/quantum-computing-exploration-for-drug-discovery-on-aws/pull/180/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='30'/>

Reply:
<p>fix bug in new commit</p>